src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -68,6 +68,20 @@ internal class BuildCommand : CommandBase
             Description = "Additional information used by this tool to generate and run build scripts.")]
         public string[] Properties { get; set; }
 
+        public static string BuildOperationName(IEnvironment env)
```

Comment:
Did we find out how it is generally recommended for this kind of use-case? Like this is pretty basic case...here they seem to be recommending custom dimensions https://stackoverflow.com/a/41705558/1184056

Reply:
Based on our offline discussion, ignore this comment